web/components/footer.js

```diff
@@ -154,14 +154,14 @@ function Footer({ children }) {
         <Link href="/docs">
           <a>Docs</a>
         </Link>
-        <a href="https://github.com/replicate/replicate">GitHub</a>
-        <a href="https://twitter.com/replicateai">Twitter</a>
-        <a href="mailto:team@replicate.ai">team@replicate.ai</a>
+        <a href="https://github.com/replicate/keepsake">GitHub</a>
+        <a href="https://twitter.com/keepsakeai">Twitter</a>
+        <a href="mailto:team@keepsake.ai">team@keepsake.ai</a>
```

Comment:
```suggestion
        <a href="https://github.com/replicate/keepsake">GitHub</a>
        <a href="https://twitter.com/replicateai">Twitter</a>
        <a href="mailto:team@replicate.ai">team@replicate.ai</a>
```

Lots of website stuff which I will fix in follow-up